docs/docs-content/vertex/system-management/enable-non-fips-settings/use-non-fips-addon-packs.md

```diff
@@ -12,79 +12,83 @@ tags: ["vertex", "non-fips"]
 
 
 Palette VerteX provides the following FIPS-compliant infrastructure components in Kubernetes clusters it deploys. Review [FIPS-Compliant Components](../../fips/fips-compliant-components.md) to learn more.
-
-<br />
-
     
 - Operating System (OS)
-
 - Kubernetes
-
 - Container Network Interface (CNI)
-
 - Container Storage Interface (CSI)
 
-As shown in the screenshot below, the FIPS-compliant icon used to indicate full FIPS compliance is displayed next to Palette VerteX infrastructure components in the cluster profile stack. To learn about other icons Palette VerteX applies, refer to [FIPS Status Icons](../../fips/fips-status-icons.md).
+VerteX also provides the flexibility to utilize non-FIPS infrastructure packs. To do this, you must add the **Non-fips pack registry** to the tenant. Our support team will provide you with the registry URL.
+
+You can allow tenant users to use non-FIPS infrastructure packs and to customize their cluster profiles by using non-FIPS add-on packs. Add-on packs enhance cluster functionality by adding profile layers such as system apps, authentication, security, monitoring, logging, and more.
```

Comment:
I think we need to change the intro to this paragraph based on the suggestion above. We can speak holistically now, versus just focusing on add-on packs.

Reply:
Rewrote.